src/fyrox/scene/light_node.md

```diff
@@ -161,17 +161,17 @@ fn use_rayleigh_scattering(scene: &mut Scene, light_handle: Handle<Node>) {
 
 ## Shadows
 
-Spot and point lights are both supports shadows, however directional light still lacks shadows. There is a
+Spot and point lights both support shadows, however directional light still lacks shadows. There is a
 [tracking issue](https://github.com/FyroxEngine/Fyrox/issues/220) for that.
 
 ## Performance
 
-Lights are not very cheap, every light source has some performance impact. As a general rule, try to keep amount
+Lights are not very cheap, every light source has some performance impact. As a general rule, try to keep the amount
 of light sources at reasonable levels and especially try to avoid creating tons of light sources in a small area.
-Keep in mind that the less area the light need to "cover", the higher the performance. This means that you can have
-tons of small light sources almost free.
+Keep in mind that the less area the light needs to "cover", the higher the performance. This means that you can have
+tons of small light sources almost freely.
```

Comment:
I think that distorts initial meaning, it should be `for free` (but I'm not sure).

Reply:
this paragraph is so confusing. 
there are other places like so as well, although grammatically OK, i didn't rephrase them because i did't know what it was about to say.